lib/ExponentialBackoff.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var ExponentialBackoff = function(minimum_backoff, exponent, starting_backoff) {
+    if (minimum_backoff === undefined) {
+        minimum_backoff = 50;
+    }
+
+    if (exponent === undefined) {
+        exponent = 2.0;
+    }
+
+    if (starting_backoff === undefined) {
+        starting_backoff = minimum_backoff;
+    }
+
```

Comment:
Oh yeah, good idea. I normally favor super explicit checking in Javascript because so many things are falsy, but in this case we don't want any of those values to be 0 anyway.